CONFIGURATION.md

```diff
@@ -11,6 +11,33 @@ Here are some Stargate-relevant property groups that are necessary for correct s
+| `stargate.jsonapi.document.limits.max-array-length`      | `int` | `100`       | The maximum length of a single array in a document.                                                         |
+
+## Operations configuration
+*Configuration for the operation execution, defined by [OperationsConfig.java](src/main/java/io/stargate/sgv2/jsonapi/config/OperationsConfig.java).*
+
+| Property                                                | Type  | Default | Description                                                                                                                                                                                         |
+|---------------------------------------------------------|-------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `stargate.jsonapi.operations.default-page-size`         | `int` | `20`    | The default Cassandra page size used for read queries.                                                                                                                                              |
+| `stargate.jsonapi.operations.default-sort-page-size`    | `int` | `100`   | The default Cassandra page size used for read queries that are used for sorting purposes.                                                                                                           |
+| `stargate.jsonapi.operations.max-limit`                 | `int` | `1000`  | // TODO from code this seems like default, not maximum???                                                                                                                                           |
+| `stargate.jsonapi.operations.max-sort-read-limit`       | `int` | `10000` | The maximum amount of documents that could be sorted using the in-memory sorting.                                                                                                                   |
+| `stargate.jsonapi.operations.max-document-insert-count` | `int` | `20`    | The maximum amount of documents that can be inserted in a single operation. The request will fail fast without inserts if limit is broken.                                                          |
+| `stargate.jsonapi.operations.max-document-update-count` | `int` | `20`    | The maximum amount of documents that can be updated in a single operation. In case there are more documents that could be updated, the operation will set the `moreData` response status to `true`. |
+| `stargate.jsonapi.operations.max-document-delete-count` | `int` | `20`    | The maximum amount of documents that can be deleted in a single operation. In case there are more documents that could be delete, the operation will set the `moreData` response status to `true`.  |
```

Comment:
@maheshrajamani could you check this, especially the `max-limit`?

Reply:
@ivansenic @maheshrajamani I think Mahesh needs to confirm this one.

Reply:
Basically this is the maximum limit of document we will return for a query. It's redundant with in memory sort (max doc read for sort is 10k) and limit option. We can evaluate the use case and remove it.

Reply:
OK I will move this to the `stargate.jsonapi.operations.max-read-limit` and try to explain it..

Reply:
Done. Removed `maxLimit` as agreed with @maheshrajamani